semcore/bulk-textarea/src/translations/en.json

```diff
@@ -0,0 +1,9 @@
+{
+  "BulkTextarea.ClearAllButton.buttonText": "Clear all",
+  "BulkTextarea.Counter.ofAllowedRows:sr-message": "of {rowsNumber} allowed rows",
+  "BulkTextarea.Counter.limitReached:sr-message": "Limit is almost reached",
+  "BulkTextarea.Counter.limitExceeded:sr-message": "Limit is exceeded",
+  "BulkTextarea.ErrorsNavigation.totalErrors": "{errorsCount} {errorsCount, plural, one {error} other {errors}}",
+  "BulkTextarea.ErrorsNavigation.selectedError": "{errorIndex} out of {errorsCount}",
+  "BulkTextarea.ErrorsNavigation.nextError:aria-label": "Go to the {index, selectordinal, one {#st} two {#nd} few {#rd} other {#th}} invalid value"
```

Comment:
Let's simplify this a little bit according to our UX writers recommendations. Let's rename buttons to "Previous error" and "Next error".

<img width="961" alt="image" src="https://github.com/user-attachments/assets/0e2983b2-9492-4e99-a282-386a86b3ede3" />